techlibs/quicklogic/synth_quicklogic.cc

```diff
@@ -316,9 +316,13 @@ struct SynthQuickLogicPass : public ScriptPass {
                 run("iopadmap -bits -outpad outpad A:P -inpad inpad Q:P -tinoutpad bipad EN:Q:A:P A:top");
             } else {
                 run("clkbufmap -buf $_BUF_ Y:A -inpad ck_buff Q:A");
```

Comment:
This pass should be removed from OpenFPGA flow as well as there are no global clock buffers in the architecture.